### PR TITLE
Support Attach subcommand in pypodman

### PR DESCRIPTION
**Pull request by jwhonce**
* Fix some random error handling

Signed-off-by: Jhon Honce <jhonce@redhat.com>

### CODE REVIEW

**Comment by jwhonce** [comment]
@baude @TomSweeneyRedHat @haircommander PTAL

**Comment by TomSweeneyRedHat** [comment]
LGTM assuming happy tests

File: contrib/python/pypodman/pypodman/lib/actions/attach_action.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+            nargs='*',
+            help='image to instantiate and attach to',
+        )
+        parser.set_defaults(class_=cls, method='attach')
+
+    def __init__(self, args):
+        """Construct Attach class."""
+        super().__init__(args)
+        if not args.image:
+            raise ValueError('You must supply at least one image id'
```
[the remark this answers is not in the document]
Comment (jwhonce) [reply contrib/python/pypodman/pypodman/lib/actions/attach_action.py]: Fixed

**Comment by rhatdan** [comment]
LGTM
@rh-atomic-bot r+

**Pull request closed without merging**